common/server_room.cpp

```diff
@@ -232,6 +234,22 @@ void Server_Room::say(const QString &userName, const QString &s, bool sendToIsl)
     event.set_name(userName.toStdString());
     event.set_message(s.toStdString());
     sendRoomEvent(prepareRoomEvent(event), sendToIsl);
+ 
+    if (chatHistorySize != 0) {
+
+        ServerInfo_ChatMessage chatMessage;
+        QDateTime dateTime = dateTime.currentDateTime();
+        QString dateTimeString = dateTime.toString("MM/dd/yy hh:mm:ss");
```

Comment:
It shouldn't be used any were else.  The MM/dd/yy format was added to show the historic date of the message for rooms that may have messages dating back in the days rather than minutes.  THe example I told @ZeldaZach was I can see a main room that may only have 20 messages that show up as history which the date wont really matter.  But in a mod only room we may  have 100+ messages all stated over a much longer day/week time span.